examples/reference-entity-model/entities/Customer.entity.json

```diff
@@ -1,98 +1,111 @@
-            "$ref": "#/definitions/Address"
-          }
-        },
-        "billing": {
-          "description": "Example of a single-value structured property",
-          "$ref": "#/definitions/Address"
-        },
-        "birthDate": {
-          "datatype": "date"
-        },
-        "status": {
-          "datatype": "string"
-        },
-        "customerSince": {
-          "datatype": "date"
-        },
-        "orders": {
-          "datatype": "array",
-          "description": "Example of a relationship to another entity type",
-          "items": {
-            "$ref": "http://example.org/Order-0.0.1/Order"
-          }
-        }
-      }
-    },
-    "Address": {
-      "required": [ ],
-      "pii": [ ],
-      "elementRangeIndex": [ ],
-      "rangeIndex": [ ],
-      "wordLexicon": [ ],
-      "properties": {
-        "street": {
-          "datatype": "array",
-          "items": {
-            "datatype": "string",
-            "collation": "http://marklogic.com/collation/codepoint"
-          }
-        },
-        "city": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "state": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "zip": {
-          "$ref": "#/definitions/Zip"
-        }
-      }
-    },
-    "Zip": {
-      "required": [ ],
-      "properties": {
-        "fiveDigit": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "plusFour": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        }
-      }
-    }
-  }
-}
+{
+  "info": {
+    "title": "Customer",
+    "version": "0.0.1",
+    "baseUri": "http://example.org/"
+  },
+  "definitions": {
+    "Customer": {
+      "required": [
+        "name"
+      ],
+      "primaryKey": "customerId",
+      "properties": {
+        "customerId": {
+          "datatype": "integer",
+          "facet": true
+        },
+        "name": {
+          "datatype": "string",
+          "facet": false,
+          "collation": "http://marklogic.com/collation/codepoint"
+        },
+        "nicknames": {
+          "datatype": "array",
+          "description": "Example of a multi-value property of simple values",
+          "facet": false,
+          "items": {
+            "datatype": "string"
+          }
+        },
+        "shipping": {
+          "datatype": "array",
+          "description": "Example of a multi-value property of structured values",
+          "facet": false,
+          "items": {
+            "$ref": "#/definitions/Address"
+          }
+        },
+        "billing": {
+          "description": "Example of a single-value structured property",
+          "$ref": "#/definitions/Address"
+        },
+        "birthDate": {
+          "datatype": "date",
+          "facet": true

```

Comment:
If the property is of type "date" and facet = true, does that translate to a date range selector in the UI, as opposed to a set of values?

Reply:
@rjrudin Ideally it should work after generating the path range indexes for facetable properties. But there is a bug currently in UI that the facets for range path indexes are not shown.
The entitySearch endpoint though is returning the response.

Reply:
Ignore my comment - I forgot this is only adjusting the modeling UI. How this translates to search options and the search UI is out of scope.